dali/test/python/test_operator_laplacian.py

```diff
@@ -0,0 +1,610 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali.pipeline import pipeline_def
+import nvidia.dali.types as types
+import nvidia.dali.fn as fn
+
+import numpy as np
+import cv2
+from scipy.ndimage import convolve1d, filters as sp_filters
+import os
+from nose_utils import assert_raises
+
+from test_utils import get_dali_extra_path, check_batch, RandomlyShapedDataIterator
+
+
+data_root = get_dali_extra_path()
+images_dir = os.path.join(data_root, 'db', 'single', 'jpeg')
+
+test_iters = 4
+min_window_size = 3
+max_window_size = 23
+
+shape_layout_axes_cases = [((20, 20, 30, 3), "DHWC", 3), ((20, 20, 30), "", 3),
+                           ((20, 30, 3), "HWC", 2), ((20, 30), "HW", 2),
+                           ((3, 30, 20), "CWH", 2), ((5, 20, 30, 3), "FHWC", 2),
+                           ((5, 10, 10, 7, 3), "FDHWC", 3),
+                           ((5, 3, 20, 30), "FCHW", 2),
+                           ((3, 5, 10, 10, 7), "CFDHW", 3)]
```

Comment:
I don't think we support such format in other operators. Do you see any value in supporting it here?

Reply:
done